community/opendkim/APKBUILD

```diff
@@ -3,55 +3,63 @@
 # Maintainer: Valery Kartel <valery.kartel@gmail.com>
 pkgname=opendkim
 pkgver=2.10.3
-pkgrel=4
+pkgrel=5
 pkgdesc="DKIM signing and verifying filter for MTAs"
 url="http://www.opendkim.org"
 arch="all"
 license="custom"
+depends=""
 makedepends="libressl-dev libmilter-dev"
 pkgusers="opendkim"
 pkggroups="opendkim"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dbg"
-install="$pkgname.pre-install"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dbg $pkgname-utils"
+install="$pkgname.pre-install $pkgname.post-install"
 source="http://downloads.sourceforge.net/project/$pkgname/$pkgname-$pkgver.tar.gz
-	opendkim.conf
-	opendkim.initd"
-builddir="$srcdir/$pkgname-$pkgver"
+	$pkgname.confd
+	$pkgname.initd"
+builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
 	cd "$builddir"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
+		--sysconfdir=/etc/$pkgname \
 		|| return 1
 	make || return 1
 }
 
 package() {
 	cd "$builddir"
-
+	# add libressl to depends. need for post-install script
+	depends="$depends libressl"
```

Comment:
really do not need it here. Both the main package and -utils subpackage depends on libressl